samples/core/kfp_env_validation/kfp_env_validation.ipynb

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
<p>Feel free to correct me if I'm wrong: can we directly passing a list of str-typed API names instead of relying on string processing?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/pipelines/pull/2843/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Lightweight components support passing in lists (they're encoded as JSON in transit).</p><p>I do not have strong opinion whether the format should be a comma-separated string or JSON array. Perhaps the former can be easier for UX users launching the pipeline as the UX does not have support for serializing (lists).</p>

Reply:
Either way works,  though I second Alexey, I think a comma separated list makes it easier for users to input in the UI.